src/maps.py

```diff
@@ -45,3 +51,20 @@
     "XXX                                                                                                                           XXX",
     "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
 ]
+
+city_map_description = (
+    "As you step out of the dark and confining room, you're greeted by "
+    "the sight of an expansive green area. The soft blades of grass tickle "
+    "your feet as you take in your surroundings. The area is dotted with tall "
+    "trees, providing shade and a sense of tranquility. However, you quickly "
+    "remember the task at hand - to find a way to survive. You'll need to explore "
+    "the area and take advantage of the resources it has to offer. The fresh air "
+    "and open space invigorate you, giving you the energy to tackle whatever "
+    "challenges lay ahead."
```

Comment:
Is this from GPT or you? It's well written.

Reply:
GPT! See the [issue](https://github.com/GuillemBarroso/CLing/issues/27#issuecomment-1411963901). No way I could write this on morphine...